regression/cbmc-java/NullPointerException3/Test.java

```diff
@@ -0,0 +1,18 @@
+class B extends RuntimeException {}
+
+class A {
```

Comment:
NPE2 uses putfield, whereas NPE3 uses getfield. They ultimately rely on the same code but I still wanted to check they are both good.